src/support/hashing.h

```diff
@@ -28,7 +28,10 @@ JL_DLLEXPORT uint32_t memhash32_seed(const char *buf, size_t n, uint32_t seed) J
 #ifdef _P64
 STATIC_INLINE uint64_t bitmix(uint64_t a, uint64_t b) JL_NOTSAFEPOINT
 {
-    return int64hash(a^bswap_64(b));
```

Comment:
Is `bswap_64` still needed after this PR, or it can be removed?

Reply:
I don't see it referenced elsewhere so I guess so